k8s/elastic-gke-logging/chart/elastic-gke-logging/templates/elasticsearch-manifests.yaml.template

```diff
@@ -90,6 +90,8 @@ spec:
           containerPort: 9200
         - name: tcp-transport
           containerPort: 9300
+        - name: exporter
```

Comment:
optional: move it sorted by containerPort

Reply:
Done.